src/main/java/com/synopsys/integration/jenkins/scan/service/scm/SCMRepositoryService.java

```diff
@@ -0,0 +1,50 @@
+package com.synopsys.integration.jenkins.scan.service.scm;
+
+import com.cloudbees.jenkins.plugins.bitbucket.BitbucketSCMSource;
+import com.synopsys.integration.jenkins.scan.exception.ScannerJenkinsException;
+import com.synopsys.integration.jenkins.scan.global.ApplicationConstants;
+import hudson.EnvVars;
+import hudson.model.TaskListener;
+import java.util.Map;
+import jenkins.model.Jenkins;
+import jenkins.scm.api.SCMSource;
+import jenkins.scm.api.SCMSourceOwner;
+
+public class SCMRepositoryService {
+    private final TaskListener listener;
+    private final EnvVars envVars;
+    private static String jobName;
+
+    public SCMRepositoryService(TaskListener listener, EnvVars envVars) {
+        this.listener = listener;
+        this.envVars = envVars;
+    }
+
+    public Object fetchSCMRepositoryDetails(Map<String, Object> scanParameters) throws ScannerJenkinsException {
+        jobName = envVars.get(ApplicationConstants.ENV_JOB_NAME_KEY)
```

Comment:
I see this variable to be unused. If there is no use of it, please remove it.

Reply:
Removed form here and added where it is used - https://github.com/synopsys-sig/synopsys-security-scan-plugin/blob/dev/src/main/java/com/synopsys/integration/jenkins/scan/service/scm/SCMRepositoryService.java#L36